diagnostic.md

```diff
@@ -98,9 +120,7 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The color yellow and font-family of sans-serif.
 
```

Comment:
Correct on color, but font family will be cursive because inline html style declarations are the most specific styling and therefore always listened to over classes